SocialCareCaseViewerApi/V1/Boundary/Response/MashReferral_2.cs

```diff
@@ -0,0 +1,29 @@
+
+using System.Collections.Generic;
+
+#nullable enable
+namespace SocialCareCaseViewerApi.V1.Boundary.Response
+{
+    public class MashReferral_2
```

Comment:
Before I dig into the rest of the code, this feels like a good moment to rename it from `MashReferral_2` to something more meaningful? Open to suggestions and I'm not sure we have a precedent to follow in these scenarios

I guess _ideally_ the entity `MashReferral` would be the same structure no matter which data source it came from, just the method for accessing it would change (e.g. `getByIdFromMongo` or `getByIdFromPostgres`).

Reply:
To give you a bit more context, the idea was to replicate the current functions adding a suffix, then, once they are ready, delete the old ones and the suffix. Hence the new files and classes. What you're suggesting makes sense too, so will discuss this with @JohnFarrellDev

Reply:
Got it – I'll take your lead on it. If you're going to keep `_2` and plan to remove the old ones (and the suffix) in the near future, as long as we have it ticketed up and prioritised soon then I'm happy 😄 it's easy for this stuff to be forgotten!